privacy_evaluator/attacks/property_inference_attack.py

```diff
@@ -1,33 +1,115 @@
-from privacy_evaluator.attacks.attack import Attack
-from privacy_evaluator.classifiers.classifier import Classifier
-import privacy_evaluator.utils.data_utils as data_utils
-from privacy_evaluator.utils.trainer import trainer
-from privacy_evaluator.models.tf.conv_net_meta_classifier import ConvNetMetaClassifier
-from privacy_evaluator.models.tf.cnn import ConvNet
+from ..attacks.attack import Attack
```

Comment:
A general comment at the beginning. Unfortunately, you did not assign me to review the pull request. Therefore, I didn't see it. In the sprint reviews, it regularly comes up that you would like to have sprints integrated earlier. To make that possible, it is necessary to contribute, e.g. in the form of letting me know that there is something (by tagging and or assigning me).

Reply:
This was my mistake. I thought by requesting your review you will get a notification. Sorry for that. 

I started to resolve your changes to update marked comments. The remaining changes will be done by tomorrow.